### ISSUE TITLE
Web: hamburger menu does not have tabIndex

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#95418]
You cannot "tab" through the web interface to eventually reach the hamburger menu because it does not have any `tabIndex` it seems.

//cc @isidorn 

Related: https://github.com/microsoft/vscode/issues/82759

**Comment by isidorn** [comment]
We should fix this asap. Assignin to May and to myself so we look into this.

**Comment by isidorn** [comment]
Ok looked a bit into this and there are a couple of more issues:

1) Hambuger menu does not belong to the composite bar and thus up / down navigation does not travel to it
2) If I make the Hamburger menu focusable via tabIndex as soon as it is focused the focus gets stuck there due to this [code](https://github.com/microsoft/vscode/blob/41329852116b5d541d5598dca9a018873ed78399/src/vs/base/browser/ui/menu/menubar.ts#L144)
3) Hamburger menu expands menu when down is pressed, it should also expand when `right` and `space` are pressed

@sbatten is it possible to maybe put this on the plan so we fix the hamburger menu with regards to these issues?

**Comment by isidorn** [comment]
SteVen informed me this will happen.
Thus unassigning myself. SteVen let me know if I can help and feel free to assign this issue to me again.

**Comment by sbatten** [comment]
@isidorn I made it accessible via tab and allow space to trigger. Because it isn't really part of the viewlet icons I did not allow right. I found that it would be confusing given pressing right on activity bar icon focuses the next icon.

**Comment by isidorn** [comment]
Thanks @sbatten I just tried it out and you can focus into it via tab. However my conern from above is still not tackled 
* Hambuger menu does not belong to the composite bar and thus up / down navigation does not travel to it

Up / down navigation should navigate to it nicely, especialy since F6 focuses the activity bar and users use up / down to navigate over actions there.
Due to this reopening this issue.

**Comment by bpasero** [comment]
Is that a requirement for all things in the activity bar? Because we have more:
* Home Indicator
* Menu
* Views
* Accounts
* Gear

**Comment by isidorn** [comment]
Well idealy up / down would navigate through all the activity bar items. So yeah the Gear would also be a part of the navgiation.

**Comment by sbatten** [comment]
Is that right? The viewlets cycle through with arrows. More often than not, I would expect users to be navigating through those. You can always get to the menu or the next icon with tab.

**Comment by sbatten** [comment]
I can see the value there, but analogously, should panel also have the right arrow move from panel titles to panel actions? Currently our action bar areas are treated as units that you must tab out.

**Comment by isidorn** [comment]
@sbatten yes I think that would be cool also in the panel.
The fact that we use differetn `actionBars` I feel is an implementation detail.

**Comment by sbatten** [comment]
If that's the case, I think this is a larger change than I anticipated and I may not be able to fit it in this milestone.

**Comment by isidorn** [comment]
Makes sense, feel free to move to June then. Thanks

**Comment by sbatten** [comment]
@isidorn I was unable to find a general solution for action bars, so instead, I looked for places where this made sense.
1. Activity Bar
2. Panel Header Area
3. Editor tab bar + actions

I tackled 1 with this issue. I will wait for your return to create separate issues for the other 2 if you are on board with them as well.
